src/app/secure-messages/secure-messages.service.ts

```diff
@@ -126,7 +126,7 @@ export class SecureMessagesService {
     public updateDraft(id: string, draftMessage: DraftMessage): Observable<any> {
 
         return this.http.put(
-            SecureMessagesService.BASE_URL + 'draft/' + id + '/modify',
+            SecureMessagesService.BASE_URL + 'save-draft',
             draftMessage,
```

Comment:
Could just use the 'saveDraft' function here as the id is never used now. i'm assuming it's getting passed in the 'draftMessage'.

Reply:
We could but it causes a chunk of tests to fail.
Since we're not planning on working any more on this repo I think it might be best to not spend time optimising this kind of thing